docs/take-to-prod.md

```diff
@@ -0,0 +1,108 @@
+Taking your Spark .Net Application to Production
```

Comment:
Do we need to call it ".NET for Apache Spark" or ".NET for Spark" application instead (since we steer away from calling it Spark.NET publicly)? Also, I think ".NET" should be all caps for consistency.

Reply:
Sure, I will change it to `.NET for Spark` for now. And keep all `.NET` caps. Thanks!